Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/FirefoxBrowserContentLocator.cs

```diff
@@ -60,13 +62,36 @@ public Rectangle GetBrowserContentBounds (IWebDriver driver)
       return result;
     }
 
+    [NotNull]
     private static AutomationElement GetFirefoxContentElement (AutomationElement firefoxWindow)
+    {
+      foreach (var _ in Enumerable.Range (0, 3))
+      {
+        try
+        {
+          var result = GetFirefoxDocumentControl (firefoxWindow);
+
+          if (result != null)
+            return result;
+        }
+        catch (ElementNotAvailableException)
+        {
+        }
+
+        Thread.Sleep (TimeSpan.FromSeconds (1));
+      }
+
+      throw new TimeoutException ("The Firefox content window could not be found within a timeout of 2 seconds.");
+    }
+
+    [CanBeNull]
+    private static AutomationElement GetFirefoxDocumentControl (AutomationElement firefoxWindow)
```

Comment:
I believe we're supposed to avoid static

Reply:
Right, I'll change both methods.

Reply:
Yep, don't use static unless you want to communicate something with it (or need it because it's called from another static member).